Core/Importers/PrivatTransactionsImporter.cs

```diff
@@ -0,0 +1,41 @@
+﻿using ExcelDataReader;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Core.Importers
+{
+    public class PrivatTransactionsImporter : TransactionsImporter
+    {
+        public override List<Transaction> Load(Stream file)
+        {
+            System.Threading.Thread.CurrentThread.CurrentCulture =
+                System.Globalization.CultureInfo.CreateSpecificCulture("en-GB");
+            Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
+            
+            //remove this
+            file = File.Open("statements.xls", FileMode.Open, FileAccess.Read);
```

Comment:
Codacy found an issue: [Introduce a new variable instead of reusing the parameter 'file'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5836065)